cocos/ui/edit-box/EditBox-mac.mm

```diff
@@ -253,13 +255,6 @@ void init(const cc::EditBox::ShowInfo &showInfo) {
     init(showInfo);
 
     EditBox::_isShown = true;
-    EditBox::mouseDownListenerId = EventDispatcher::addCustomEventListener(EVENT_MOUSE_DOWN, EditBox::onMouseDown);
-}
```

Comment:
no  need  to register the mouse  event any more

Reply:
Why no need it now?

Reply:
Got it, end editing event will cover it.